forum/investors/models.py

```diff
@@ -52,4 +56,34 @@ class Meta:
         verbose_name_plural = 'Investors'
 
     def __str__(self):
-        return self.company_name
\ No newline at end of file
+        return self.company_name
+
+
+class InvestorStartup(models.Model):
```

Comment:
The model name`InvestorStartup` is descriptive, but you might consider renaming it to something more domain-specific, such as `Follow` or `InvestorFollow`, since it represents a relationship of following between investors and startups.